### ISSUE TITLE
Color Contrast on CTA Buttons

### ISSUE THREAD

**Issue by courtney-heitman** [w3c/silver#268]
I was experimenting with the APCA color contrast standard and noticed that all of my [company's Call to Action (CTA) button](https://www.myndex.com/APCA/?BG=61c250&TXT=000000&DEV=d12dSmthScl&BUF=APCA-D), which are currently compliant, will fall into the 1 rating. 

So I messed around with some other companies colors, [the Github green buttons](https://www.myndex.com/APCA/?BG=e0001b&TXT=ffffff&DEV=d12dSmthScl&BUF=APCA-D), also get a 1. 

[The purple Material design buttons](https://www.myndex.com/APCA/?BG=6200ee&TXT=ffffff&DEV=d12dSmthScl&BUF=APCA-D) get a 2. 

It looks like any primary brand color that is a brighter/vibrant color is going to have a hard time getting their call to action buttons above a 1 or 2 on the rating. How do we think that will effect the ratings for conformance? There's lots of marketing pages on the web that have multiple CTA buttons on them.

**Comment by Myndex** [comment]
Hi @courtney-heitman   can you show me an image, screenshot, or link?

Only the TEXT on the button is required to be at the level shown in the APCA tool, and in fact the APCA tool is calibrated for text that should be "fluently readable", emphasizing body text etc.

"Spot reading" and  non-text elements have a much wider degree of acceptable contrast levels. This is "sort of" shown, with the **[ N ]** symbol in the font grid, and the manual lookup table's "©§™" 

ALSO: highly saturated colors can have a negative effect on readability. 

**Nevertheless, I would like to examine any examples you have.** This is particularly important right now as we just rescaled the lookup table in conjunction with modifying the exponent constants, and I want to see any edge cases that seem out of place, if any do exist.

Thank you

Andy

**Comment by Myndex** [comment]
Hi @courtney-heitman 

DOH to me, I just realized you had links in your post.

I'm not sure why you think these buttons all get a 1 or 2 — they don't. They all get a 4 for a typical CTA font size and weight. Can y9ou please confirm the colors below are indeed the colors you are talking about?

**1) The worst of the bunch** is the green/black (the link you said is your company?? `#000` text and `#61c250`BG) . This requires a 23.5px normal weight or 14px bold font. almost exactly the same as WCAG 2 at 4.5:1, in fact slightly more relaxed. Now as it happens, WCAG 2 math claims this is a color at over 9.3:1, and that is incorrect, nevertheless, the color pair passes with a reasonable font size.
<img width="1014" alt="Screen Shot 2021-02-04 at 5 00 41 AM" src="https://user-images.githubusercontent.com/42009457/106896249-29fd4300-66a6-11eb-9ecf-213b32e6e810.png">


**2) The second link** is a white on red, and EASILY passes at level 4 (`#fff` and `#e0001b`) WCAG 2 says 5:1, and APCA indicates higher, as it is.
<img width="1014" alt="Screen Shot 2021-02-04 at 5 01 06 AM" src="https://user-images.githubusercontent.com/42009457/106896327-44372100-66a6-11eb-8ac8-fcb2f0deb8b4.png">


**3) The third is a white on blue** (`#fff` and `#6200ee`), and is literally near maximum contrast.
<img width="1012" alt="Screen Shot 2021-02-04 at 5 00 53 AM" src="https://user-images.githubusercontent.com/42009457/106896352-4e591f80-66a6-11eb-9521-eda86db85c51.png">


If you are led to believe these are score 1, then I have not explained how this works properly at all. This is a massive concern for me, as I do want to be clear and easy to understand.

**Can you please describe to me in as much detail as possible why you feel these are all score level 1?** Because I am confused a bit here... If you have samples of the _ACTUAL_ buttons, that would be extremely helpful.

THANK YOU!!!

Andy

**Comment by Myndex** [comment]
Hi @courtney-heitman 
TO ADD: I think I found the site you mentioned that `#60c251` is the color of the CTA button. The failure is not the color, but the very small thin font size, at 11px & 400 wgt. This is too small for content text, and fails even score level 1 the way the lookup table is presently scaled.

HOWEVER, as I said, the lookup tables are undergoing some revisions. I want you to know I really appreciate this feedback

Thank you 

Andy

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by ChrisLoiselle** [comment]
@jspellman , this is ready for survey. Per @Myndex / Andy's response on Feb 4th, " I think I found the site you mentioned that #60c251 is the color of the CTA button. The failure is not the color, but the very small thin font size, at 11px & 400 wgt. This is too small for content text, and fails even score level 1 the way the lookup table is presently scaled."

**Comment by ChrisLoiselle** [comment]
DRAFT RESPONSE: We believe the questions have been answered by Andy per https://github.com/w3c/silver/issues/268#issuecomment-773756299 . Per Andy " I think I found the site you mentioned that #60c251 is the color of the CTA button. The failure is not the color, but the very small thin font size, at 11px & 400 wgt. This is too small for content text, and fails even score level 1 the way the lookup table is presently scaled."

I hope this answers your question, if it does not, please feel free to follow-up.

Thank you,
Chris

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
